light_curves/light_curve_generator.md

```diff
@@ -80,19 +80,19 @@ from astropy.table import Table
 # local code imports
 sys.path.append('code_src/')
 from data_structures import MultiIndexDFObject
-from gaia_functions import Gaia_get_lightcurve
-from HCV_functions import HCV_get_lightcurves
-from heasarc_functions import HEASARC_get_lightcurves
-from icecube_functions import Icecube_get_lightcurve
-from panstarrs import Panstarrs_get_lightcurves
+from gaia_functions import gaia_get_lightcurves
+from hcv_functions import hcv_get_lightcurves
+from heasarc_functions import heasarc_get_lightcurves
+from icecube_functions import icecube_get_lightcurves
+from panstarrs_functions import panstarrs_get_lightcurves
 from plot_functions import create_figures
 from sample_selection import (clean_sample, get_green_sample, get_hon_sample, get_lamassa_sample, get_lopeznavas_sample,
-    get_lyu_sample, get_macleod16_sample, get_macleod19_sample, get_ruan_sample, get_SDSS_sample, get_sheng_sample, get_yang_sample)
-from TESS_Kepler_functions import TESS_Kepler_get_lightcurves
+    get_lyu_sample, get_macleod16_sample, get_macleod19_sample, get_ruan_sample, get_sdss_sample, get_sheng_sample, get_yang_sample)
+from tess_kepler_functions import tess_kepler_get_lightcurves
 # Note: WISE and ZTF data are temporarily located in a non-public AWS S3 bucket. It is automatically
 # available from the Fornax SMCE, but will require user credentials for access outside the SMCE.
-from WISE_functions import WISE_get_lightcurves
-from ztf_functions import ZTF_get_lightcurve
+from wise_functions import wise_get_lightcurves
```

Comment:
I don't care about capitalization (which is potentially why the code needed this sort of PR), but I don't know if others care about capitalization.  For example, WISE is capitalized because it is an acronym, so are we doing something grammatically incorrect (or worse, offending someone) by changing it to lower case?  Same for the other telescope names.

Reply:
In python it is pretty standard to name functions and modules using snake case (including lower case). By contrast, classes typically use camel case.

Reply:
#215 really benefits from these using a uniform syntax, including the case. It could deal with something non-uniform but would require more code and/or rely on the user to enter the name exactly right including case.